src/java/org/apache/cassandra/service/accord/IAccordService.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.service.accord;
+
+import accord.messages.Request;
+import accord.primitives.Txn;
+import accord.topology.TopologyManager;
+import org.apache.cassandra.db.ConsistencyLevel;
+import org.apache.cassandra.net.IVerbHandler;
+import org.apache.cassandra.service.accord.txn.TxnData;
+
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public interface IAccordService
+{
+    IVerbHandler<? extends Request> verbHandler();
+
+    void createEpochFromConfigUnsafe();
+
+    TxnData coordinate(Txn txn, ConsistencyLevel consistencyLevel);
+
+    long currentEpoch();
+
+    void setCacheSize(long kb);
+
+    TopologyManager topology();
```

Comment:
I kinda wish we could just return `Node`, but since that is what owns the threads... I doubt we can =(